osgp/platform/osgp-adapter-domain-smartmetering/src/main/java/org/opensmartgridplatform/adapter/domain/smartmetering/application/mapping/MonitoringMapper.java

```diff
@@ -36,6 +38,8 @@ public final void configure(final MapperFactory mapperFactory) {
         converterFactory.registerConverter(new PeriodicMeterReadsResponseItemDtoConverter(mapperFactory));
         converterFactory.registerConverter(new MeterReadsResponseItemDtoConverter(mapperFactory));
         converterFactory.registerConverter(new GetPowerQualityProfileDtoConverter(mapperFactory));
+        converterFactory.registerConverter(new ActualPowerQualityDtoConverter(mapperFactory));
+        converterFactory.registerConverter(new ActualValueConverter());
```

Comment:
what is Value? the name seems to generic.

Reply:
ActualValueConverter converts ActualValue like ActualPowerQualityDtoConverter converts ActualPowerQualityDto